.github/workflows/gcc.yml

```diff
@@ -40,8 +40,8 @@ jobs:
         ctest --output-on-failure
 
   # Build libamrex and all tests
-  tests_build:
-    name: GNU@7.5 C++14 Debug Fortran [tests]
+  tests_build_3D:
+    name: GNU@7.5 C++14 3D Debug Fortran [tests]
     runs-on: ubuntu-18.04
     env: {CXXFLAGS: "-fno-operator-names -Werror -Wall -Wextra -Wpedantic -Wnull-dereference -Wfloat-conversion -Wshadow -Woverloaded-virtual -O1"}
```

Comment:
I don't see a `-g` here.

Also, the recommendation from GCC for "optimize without impeding debugging" is `-Og`

Reply:
Oh, I see, it's a CMake build, in Debug mode.

Reply:
Yep, that's added in `-DCMAKE_BUILD_TYPE=Debug`